workers/base_task.py

```diff
@@ -75,6 +76,14 @@ class BaseTask(Task):
     work_path = None
     log = logging.getLogger(__name__)
 
+    label = 'No label set for worker type'
+    description = 'No description set for worker type'
```

Comment:
Wäre es nicht sinnvoll, in `__init__()` oder `run()` abzufragen, ob label und description gesetzt wurden und andernfalls einen Error zu werfen? Dann würde man verhindern, dass jemals Tasks ohne die beiden Felder implementiert werden.

Reply:
Habe es umgestellt, ist direkt ein Fehler aufgetaucht weil ich einen Schreibfehler im Variablennamen hatte. :+1: